ipykernel/debugger.py

```diff
@@ -562,6 +564,24 @@ async def richInspectVariables(self, message):
         reply["success"] = True
         return reply
 
+    async def modules(self, message):        
+        modules = [sys.modules[name] for name in sys.modules]
+        mods = []
+        for module in modules:
+            m = str(module)
+            if ".py'" in m:
+                a = {}
+                x = re.findall(r"'(.*?)'", m)
+                a[x[0]] = x[1]
+                mods.append(a)
+        reply = {
+            'body': {
+                'modules': mods,
+                'totalModules': len(modules)
```

Comment:
I think we the debugger should handle the `startModule` and `moduleCount` arguments speficied in the [DAP](https://microsoft.github.io/debug-adapter-protocol/specification#Requests_Modules) and filter the result accordingly.

Reply:
I have implemented the `startModule` and `moduleCount` in 95711b80b47537927bb2359daf43ea39cf2f0ad2